code/modules/mob/living/carbon/species.dm

```diff
@@ -262,6 +265,24 @@ var/global/list/whitelisted_species = list("Human")
 
 /datum/species/proc/equip(var/mob/living/carbon/human/H)
 
+/datum/species/proc/get_inventory_offsets()	//This is what you override if you want to give your species unique inventory offsets.
+	return list(
```

Comment:
Store this list in a **`/static`** variable before returning it. That way it'll still be species-overridable, but still be static.

Reply:
Done.